sphinx/builders/_epub_base.py

```diff
@@ -108,8 +108,8 @@ class NavPoint(NamedTuple):
 
 def sphinx_smarty_pants(t: str, language: str = 'en') -> str:
     t = t.replace('&quot;', '"')
-    t = smartquotes.educateDashesOldSchool(t)
-    t = smartquotes.educateQuotes(t, language)
+    t = smartquotes.educateDashesOldSchool(t)  # type: ignore[no-untyped-call]
```

Comment:
Could we try ignoring `untyped-call` on a [per-module](https://mypy.readthedocs.io/en/stable/config_file.html#per-module-and-global-options) basis?  (it seems like we're not quite ready for that check on these calls)